packages/components/src/components/select-combobox/__tests__/select-combobox.tests.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import { Default, DefaultOpen, EmptyValue } from "../stories/index.stories";
```

Comment:
Thanks for adding tests! Could you add one more that tests the clearing feature that you just added?

Reply:
🤦

Reply:
Done!